e2e/system/build_from_git_test.go

```diff
@@ -41,51 +50,79 @@ func (s *Suite) TestBuildFromGit() {
 	s.T().Logf("Starting instance")
 	s.Require().NoError(target.Execution().Start(ctx))
 
-	s.T().Logf("Instance started")
+	s.T().Log("Instance started")
 
+	s.T().Log("Getting file bytes")
 	// The file is created by the dockerfile in the repo,
 	// so to make sure it is built correctly, we check the file
 	data, err := target.Storage().GetFileBytes(ctx, "/test.txt")
 	s.Require().NoError(err)
 
 	data = []byte(strings.TrimSpace(string(data)))
-	s.Assert().Equal([]byte("Hello, World!"), data, "File bytes do not match")
+	if !bytes.Equal([]byte("Hello, World!"), data) {
+		s.Require().NoError(err, "File bytes do not match. Expected 'Hello, World!', got '%s'", string(data))
+	}
```

Comment:
This should be reverted. The Assert statement was handling the byte comparison. In this set up, we are actually completely ignoring the byte mismatch because if you get to this point, the `err` is nil because of the check above. So this wouldn't stop the test.

Reply:
not resolved